src/symbolic_utilities.jl

```diff
@@ -435,27 +460,48 @@ function get_argument(eqs, _indvars::Array, _depvars::Array)
     get_argument(eqs, dict_indvars, dict_depvars)
 end
 function get_argument(eqs, dict_indvars, dict_depvars)
+    "Equations, as expressions"
     exprs = toexpr.(eqs)
-    vars = map(exprs) do expr
+    "Instances of each dependent variable that appears in the expression, by dependent variable, by equation"
+    vars = map(exprs) do expr # For each equation,...
+        "Arrays of instances of each dependent variable, by dependent variable"
         _vars = map(depvar -> find_thing_in_expr(expr, depvar), collect(keys(dict_depvars)))
+        "Arrays of instances of each dependent variable that appears in the expression, by dependent variable"
         f_vars = filter(x -> !isempty(x), _vars)
-        map(x -> first(x), f_vars)
     end
+    #    vars = [depvar for expr in vars for depvar in expr ]
     args_ = map(vars) do _vars
-        ind_args_ = map(var -> var.args[2:end], _vars)
+        "Arguments of all instances of dependent variable, by instance, by dependent variable"
+        ind_args_ = map.(var -> var.args[2:end], _vars)
         syms = Set{Symbol}()
-        filter(vcat(ind_args_...)) do ind_arg
+        "All arguments in any instance of a dependent variable"
```

Comment:
are these supposed to be comments?

Reply:
Yes, I think I misread the Julia documentation on docstrings. I'm inferring you only use them for explaining what functions are/do and not what a variable is/how it's organized. I've changed them to comments, which should help with code coverage, since these lines with just strings were all showing up as not running.